### PR TITLE
Initialize repo

### PR DESCRIPTION
**Pull request by sbidoul**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you please add a README.md with at least flags, a brief description, and OCA disclaimer?

You can take this one as example: https://github.com/OCA/bank-payment/blob/8.0/README.md

**Comment by sbidoul** [comment]
@pedrobaeza done

File: README.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
+[![Build Status](https://travis-ci.org/OCA/bank-statement-import.svg?branch=8.0)](https://travis-ci.org/OCA/bank-statement-import)
+[![Coverage Status](https://coveralls.io/repos/OCA/bank-statement-import/badge.png?branch=8.0)](https://coveralls.io/r/OCA/bank-statement-import?branch=8.0)
+
 Odoo bank importation
```
Comment (pedrobaeza) [inline README.md]: I would put here `OCA modules for bank importation for Odoo` for trying to give brand to OCA.

**Comment by sbidoul** [comment]
@pedrobaeza I improved the README

**Comment by pedrobaeza** [comment]
Thanks for the change!

Another thing: @gurneyalex is reporting in other projects that `UNIT_TEST="1"` is very slow, taking 30 min or more, so we should considered its inclussion.

**Comment by sbidoul** [comment]
@pedrobaeza right. I never really understood this UNIT_TEST thing anyway, and I prefer that we keep modules and tests fully compatible. Should be ok now.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by pedrobaeza** [comment]
The goal of UNIT_TEST is to tests isolated each module.

**Comment by guewen** [comment]
:+1:

**Pull request merged**